frontend/src/component/project/Project/ProjectDoraMetrics/ProjectDoraFeedback/ProjectDoraFeedback.tsx

```diff
@@ -0,0 +1,158 @@
+}));
+
+const StyledDivider = styled(Divider)(({ theme }) => ({
+    marginTop: theme.spacing(3),
+    marginBottom: theme.spacing(1.5),
+}));
+
+const StyledFlexBox = styled(Box)(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    marginTop: theme.spacing(1),
+    marginRight: theme.spacing(3),
+}));
+
+const StyledIconWrapper = styled(Box)(({ theme }) => ({
+    color: theme.palette.primary.main,
+    marginRight: theme.spacing(1),
+    display: 'flex',
+    alignItems: 'center',
+}));
+
+const StyledHeader = styled(Typography)(({ theme }) => ({
+    fontSize: theme.fontSizes.mainHeader,
+    marginBottom: theme.spacing(2),
+    fontWeight: 'bold',
+}));
+
+const StyledLink = styled('a')(({ theme }) => ({
+    textDecoration: 'none',
+}));
+
+export const ProjectDoraFeedback = () => {
+    const { trackEvent } = usePlausibleTracker();
+    const { value, setValue } = createLocalStorage(
+        `project:metrics:plausible`,
+        { sent: false }
+    );
+    const [sentObj, setSentObj] = useState(value);
+
+    useEffect(() => {
+        setValue(sentObj);
+    }, [sentObj]);
+
+    const onBtnClick = (type: string) => {
+        try {
+            trackEvent('project-metrics', {
+                props: {
+                    eventType: type,
+                },
+            });
+
+            if (!Boolean(sentObj.sent)) {
+                setSentObj({ sent: true });
+            }
+        } catch (e) {
+            console.error('error sending metrics');
+        }
+    };
+
+    const recipientEmail = 'recipient@example.com';
+    const emailSubject = "I'd like to get involved";
+    const emailBody = `Hello Unleash,\n\nI just saw the new metrics page you are experimenting with in Unleash. I'd like to be involved in user tests and give my feedback on this feature.\n\nRegards,\n`;
+
+    const mailtoURL = `mailto:${recipientEmail}?subject=${encodeURIComponent(
+        emailSubject
+    )}&body=${encodeURIComponent(emailBody)}`;
+
+    return (
+        <StyledOuterContainer>
+            <StyledHeader variant="h1">
+                We are trying something experimental!
+            </StyledHeader>
+            <Typography>
+                We are considering adding project metrics to see how a project
+                performs. As a first step, we have added a lead time for changes
+                indicator that is calculated per feature toggle based on the
+                creation of the feature toggle and when it was first turned on
+                in an environment of type production.
+            </Typography>
+
+            <ConditionallyRender
+                condition={!sentObj.sent}
+                show={
+                    <StyledBtnContainer>
+                        {' '}
+                        <Typography>Is this useful to you?</Typography>
+                        <StyledBtnContainer>
+                            <Button
+                                sx={{ marginRight: '0.5rem' }}
```

Comment:
We moved away from using rems, but use theme.spacing instead.

Reply:
Nice catch, I thought I removed all of those. Will fix.